skins/item/color/yellow.ess

```diff
@@ -0,0 +1,76 @@
+background = __.color.yellow.light
+color hsla(49.6, 73.1%, 26.3%, 1)
```

Comment:
Unless this color will only be used here. If so, we should make the color as a variable at the top of the file.